raspberry/checkout/checkout_main.py

```diff
@@ -62,16 +66,42 @@ def process_rfid_card(self, uid):
             print("Item already scanned")
             self.interactions.buzz()
             self.interactions.indicate_error()
+
+    def cancel_action(self):
+        if self.state == 1:
```

Comment:
```suggestion
        if self.state == 0:
```
cancellujemy cały checkout w momencie skanowania. A jak jesteśmy w podsumowaniu, to czerwony usuwa produkt, czyli dokładnie na odwrót